main.py

```diff
@@ -1,11 +1,53 @@
 import init_django_orm  # noqa: F401
 
+import json
+
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
 
+    with open("players.json", "r") as f:
+        data = json.load(f)
+
+    for name, data_info in data.items():
+
+        race_name = data_info["race"]["name"]
+        race_note = data_info["race"]["description"]
+        if not Race.objects.filter(name=data_info["race"]["name"]).exists():
```

Comment:
you have variable race_name = data_info["race"]["name"] but use name=data_info["race"]["name"] inside your filter